src/2048-expert/index.html

```diff
@@ -13,6 +13,22 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+        <ul>
+            <li>Take actions with arrow keys</li>
+            <li>Start a new game by pressing enter</li>
+            <li> Tiles with the same value will get merged after moves</li>
+            <br>When a tile has the value <strong>2048.</strong>
+            <br> Congradulations! <strong>You Win!</strong>
+            <br>
+            <br>The browswer will key your best try. Start a new game and challenge your limits!
```

Comment:
Just `Start a new game and challenge your limits!` is good. Tracking the high score is more "advanced" information that can be left out of the help dialogue to keep it short and simple.